libs/utils/wa_results_collector.py

```diff
@@ -458,11 +463,17 @@ def _get_extra_job_metrics(self, job_dir, workload):
         # think we can simplify this.
         for artifact_name, path in artifacts.iteritems():
             if os.stat(path).st_size == 0:
-                self._log.info(" no data for %s",  path)
+                #self._log.info(" no data for %s",  path)
```

Comment:
Don't we want to keep that log here ? If we don't, the commented line can be removed.

Reply:
yes probably.
I have commented for testing because it was generating a lot of log when you parse a wa directory but you aborted WA after few tests